include/zephyr/bluetooth/audio/bap.h

```diff
@@ -2042,14 +2042,28 @@ int bt_bap_broadcast_sink_delete(struct bt_bap_broadcast_sink *sink);
 /** @} */ /* End of group bt_bap_broadcast_sink */
 
 /**
- * @brief Register the callbacks for the Basic Audio Profile Scan Delegator
+ * @brief Register the Basic Audio Profile Scan Delegator and BASS.
  *
+ * Register the scan deligator and Broadcast Audio Scan Service (BASS)
+ * dynamically at runtime.
+ *
+ * @param cb Pointer to the callback struct
  * Only one set of callbacks can be registered at any one time, and calling this function multiple
  * times will override any previously registered callbacks.
  *
- * @param cb Pointer to the callback struct
+ * @return int   Error value. 0 on success, errno on fail.
+ */
+int bt_bap_scan_delegator_register(struct bt_bap_scan_delegator_cb *cb);
+
+/**
+ * @brief unregister the Basic Audio Profile Scan Delegator and BASS.
+ *
+ * Unregister the scan deligator and Broadcast Audio Scan Service (BASS)
+ * dynamically at runtime.
+ *
+ * @return int   Error value. 0 on success, errno on fail.
```

Comment:
```suggestion
 * @return 0 in case of success or negative value in case of error.
```

Mainly for consistency across LE Audio.
I have to admit that I don't know if Zephyr dictates this, but I have not seen the return type attached after @return in this repo yet.

Reply:
OK. I just copied this line from the description of the function right after my added functions!
But anyway, I updated it.

Reply:
Intresting :) Anyway, lets be happy we got closer to consistency now :D